CHANGELOG.md

```diff
@@ -7,6 +7,13 @@
 + Fix a bug where zip files with directories could not be imported. 
   For example a zip with `a.wdl` and `b.wdl` could be imported but one with `sub_workflows/a.wdl` 
   and `imports/b.wdl` could not.
+
+### Migration from Google Cloud Genomics v2alpha1 to Google Cloud Life Sciences v2beta
```

Comment:
```suggestion
### Adding support for Google Cloud Genomics v2beta
```
The "m" word (migration) scares the crap out of certain users and may lead to FUD & support questions around this version of Cromwell.

Reply:
Ok, but should we keep calling it Genomics instead of Life Sciences?

Reply:
Good catch, that's my mistake authoring the suggestion. "Google Cloud Life Sciences v2beta" is definitely right.